gdc_client/settings/parser.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from functools import partial
+
+from gdc_client.common.config import GDCClientConfigShared
+
+
+log = logging.getLogger('gdc-settings')
+
+
+class SettingsResolver(object):
+    def __init__(self):
+        self.config = GDCClientConfigShared()
+
+    def ls(self, section, args):
+        log.info(self.config.to_display_string(section))
+        return self.config.to_display_string(section)
+
+    def list(self, section, args):
```

Comment:
I get that the function name is linked to the config option, but is there any way to make a special case for this `list` command? I feel funny whenever I see the clobbering of a python keyword.

Reply:
yeah, maybe `show`? 😄